k8s/yaml/deployment.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ts-bridge
+spec:
+  selector:
+    matchLabels:
+      app: ts-bridge
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: ts-bridge
+    spec:
+      securityContext:
+        runAsUser: 1000
+        runAsGroup: 1000
+        fsGroup: 1000
+      containers:
+        - image: REPLACE_ME
+          imagePullPolicy: Always
+          name: ts-bridge
+          env:
+            # Use secret in real usage
+            - name: GOOGLE_APPLICATION_CREDENTIALS
+              value: /etc/gcp/google_key.json
+          args:
+            - --storage-engine=boltdb
+            - --enable-status-page
+            - --stats-sd-project=REPLACE_ME
+            - --update-parallelism=4
+            - --sync-period=10s
+          ports:
+            - containerPort: 8080
+              name: ts-bridge
+          resources:
+            limits:
+              memory: "1Gi"
```

Comment:
Have you tested this to use between 1/2 and 1Gb RAM?

Interesting to note the k8s metric server won't expose correct usage for a job unless it has memory AND cpu resource constraints.

Reply:
I'll remove the constraint for now, pending https://github.com/google/ts-bridge/issues/52